tests/zap/test_access_and_admin_functions.py

```diff
@@ -9,29 +9,13 @@
     CVXCRV_DEPOSIT,
     VOTIUM_REGISTRY,
     NSBT,
-    CVXCRV_TOKEN,
-    CVXCRV_REWARDS,
+    DUMMY_PROOF,
+    OUTPUT_TOKEN_LENGTH,
+    FXS,
+    MAX_UINT256,
 )
 
 
```

Comment:
Test on line 37 is duplicate of that on line 32

Reply:
nice catch, will fix

Reply:
fixed in 2ff43b7952f427456e3814054bbf64e79b3feaaf